src/databricks/labs/lsql/dashboards.py

```diff
@@ -45,6 +49,38 @@ def as_dict(self) -> dict[str, str]:
         return dataclasses.asdict(self)
 
 
+@dataclass
+class WidgetMetadata:
+    order: int
+    width: int
+    height: int
```

Comment:
if widgets are optional, they have to get a default value. or `| None = None`

Reply:
Yes, currently it is not required yet as the default options are provided when creating the fallback metadata